e2e/core/settings-component.e2e.ts

```diff
@@ -0,0 +1,248 @@
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).not.toEqual('ECM', 'The Settings changes are saved');
+            expect(settingsPage.getBpmHostUrl()).not.toEqual('http://adfdev.envalfresco1.com', 'The Settings changes are saved');
+
+        });
+
+        it('[C291948] Should save ALL Settings changes when User clicks Apply button', () => {
+            settingsPage.setProviderEcmBpm();
+            loginPage.waitForElements();
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).toEqual('ALL', 'The Settings changes are not saved');
+            expect(settingsPage.getBpmHostUrl()).toEqual(TestConfig.adf.url, 'The BPM Settings changes are not saved');
+            expect(settingsPage.getEcmHostUrl()).toEqual(TestConfig.adf.url, 'The ECM Settings changes are not saved');
+
+        });
+
+        it('[C291949] Should have field validation for Content Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.clearContentServicesURL();
+            settingsPage.ecmText.sendKeys(protractor.Key.TAB);
+            settingsPage.checkValidationMessageIsDisplayed();
+            settingsPage.checkApplyButtonIsDisabled();
+        });
+
+        it('[C291950] Should have field validation for Process Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.clearProcessServicesURL();
+            settingsPage.bpmText.sendKeys(protractor.Key.TAB);
+            settingsPage.checkValidationMessageIsDisplayed();
+            settingsPage.checkApplyButtonIsDisabled();
+        });
+
+        it('[C291951] Should not be able to sign in with invalid Content Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmOption(), 'ECM');
+            settingsPage.setContentServicesURL('http://localhost:7070');
+            settingsPage.clickApply();
+            loginPage.waitForElements();
+            loginPage.enterUsername(adminUserModel.id);
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            expect(loginPage.getLoginError()).toMatch(loginError);
+        });
+
+        it('[C291952] Should not be able to sign in with invalid Process Services Url', () => {
+            settingsPage.setProvider(settingsPage.getBpmOption(), 'BPM');
+            settingsPage.setProcessServicesURL('http://localhost:7070');
+            settingsPage.clickApply();
+            loginPage.waitForElements();
+            loginPage.enterUsername(adminUserModel.id);
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            expect(loginPage.getLoginError()).toMatch(loginError);
+        });
+    });
+
+    describe('Settings Component - BPM Basic Authentication', () => {
```

Comment:
Maybe change the name of this describe to "Basic Authentication" since you are also testing for ECM, not only BPM

Reply:
done